projects/angular/src/data/datagrid/datagrid-action-overflow.ts

```diff
@@ -87,18 +100,40 @@ export class ClrDatagridActionOverflow implements OnDestroy {
     this.subscriptions.forEach(sub => sub.unsubscribe());
   }
 
+  @HostListener('window:keydown', ['$event'])
+  onKeydown($event: KeyboardEvent) {
+    if (this.keyManager) {
+      this.keyManager?.onKeydown($event);
+    }
+  }
+
   closeOverflowContent(event: Event): void {
     this.smartToggleService.toggleWithEvent(event);
   }
 
-  private focusFirstButton(): void {
+  private initalizeFocus(): void {
     if (isPlatformBrowser(this.platformId)) {
       this.zone.runOutsideAngular(() => {
         setTimeout(() => {
-          const firstButton: HTMLButtonElement | null = document.querySelector('button.action-item');
-          if (firstButton) {
-            firstButton.focus();
-          }
+          const buttons = Array.from(document.querySelectorAll<HTMLButtonElement>('button.action-item'));
```

Comment:
We're projecting with `ng-content`, so it can be anything. Even if we assume it would be only actionable elements, it may be also anchor elements, not only buttons

Reply:
I agree that this should probably be improved, but I think it's out of scope here.